nengo/utils/ipython.py

```diff
@@ -55,21 +24,15 @@
     if IPython.version_info[0] <= 2:
```

Comment:
Given we're only supporting versions 4 and greater with this PR, do we still need to try and meet all these cases?

Reply:
Yeah we should probably go through and remove these old things ... just didn't want to risk breaking anything (and also lazy), but I think it's worth doing.

Reply:
Just to be clear, is it worth doing in this PR or in a different PR?

Reply:
I'm happy to do it in this PR :man_shrugging:

Reply:
Do you want me to make the fixup commit or are you already doing it?

Reply:
I'm on it